framework/src/modules/http_api/controllers/accounts.js

```diff
@@ -52,6 +53,10 @@ function accountFormatter(totalSupply, account) {
 		);
 	}
 
+	if (formattedAccount.keys.numberOfSignatures === 0) {
```

Comment:
Maybe better leave it empty? That way any one consuming this does not need to handle the missing key and instead decide what to do with an empty object

Reply:
🤔 
if we send empty then all the account would look like 
```
{
  mandatoryKey: [
  ],
  optionalKey: [
   ],
  numberOfSignature: 0  
}
```
And this looks odd.. its like if a user is not delegate we dont send `delegate:{}` info.. so similarly if an account is not multi signature we dont send `keys:{}`.

Reply:
Oh ok if that's the pattern used it's fine then.